crates/turbopack-node/src/pool.rs

```diff
@@ -176,18 +176,21 @@ impl NodeJsPoolProcess {
                         match status {
                             Ok(status) => {
                                 let (stdout, stderr) = get_output(&mut child).await?;
+                                println!("node process exited before we could connect to it with {status}Process output:\n{stdout}\nProcess error output:\n{stderr}");
```

Comment:
I expect this was for debugging? Remove it or add it as an `eprintln`. Also, a `\n` disappeared between `{status}` and `Process output`